modules/ocf_printhost/files/enforcer

```diff
@@ -35,6 +37,21 @@ CONF_FILE = '/opt/share/enforcer/enforcer.conf'
 
 Message = namedtuple('Message', ['subject', 'body'])
 
+# Print Job Statuses:
+# 0 - Completed
+# 1 - Pending
+# 2 - Error (problematic file)
+# 3 - Error (insufficient quota)
+# 4 - Error (job processing)
```

Comment:
what does this mean?

Reply:
It’s the numerical representation for status codes

Reply:
Yes but what does `job processing` mean in relation to `JOB_ERROR`?

Reply:
Like, an error processing the job